.github/workflows/build-binaries.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.20.3
+          go-version: 1.21
```

Comment:
Could give it the path to the `go.mod` so we don't need to change it ([ref](https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file))?

Reply:
Added `go-version-file`, thanks 👍